### DISCUSSION TITLE
After the last update, I am no longer able to use cookies in insomnia.

### DISCUSSION THREAD

**Discussion by davidkukert** [Kong/insomnia#7605] [Help]
After the last update, I am no longer able to use cookies in insomnia.
It keeps giving a rendering error.
![insomnia-error](https://github.com/Kong/insomnia/assets/29695875/f16bbbf4-fabd-44cf-a0b6-b358f729a187)

**Comment by subnetmarco** [comment]
We are working on this regression, tracking it here: https://github.com/Kong/insomnia/issues/7604
